examples/usda-cdl-cultivated/collection.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "Collection",
+  "id": "usda-cdl-cultivated",
+  "stac_version": "1.0.0",
+  "description": "The UDSA CDL Cultivated Layer is based on five years (2017-2021).",
+  "links": [
+    {
+      "rel": "root",
+      "href": "../catalog.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "about",
+      "href": "https://www.nass.usda.gov/Research_and_Science/Cropland/SARS1a.php",
+      "title": "Product Landing Page"
+    },
+    {
+      "rel": "item",
+      "href": "./ancillary_cultivated_2021/ancillary_cultivated_2021.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "parent",
+      "href": "../catalog.json",
+      "type": "application/json"
+    }
+  ],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/classification/v1.1.0/schema.json"
+  ],
+  "item_assets": {
+    "cultivated": {
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "roles": [
+        "data"
+      ]
+    }
```

Comment:
We should include the classification information in the item assets (this goes for all three collection types).